### PR TITLE
Makefile: fix targets to pass tests

### PR DESCRIPTION
**Pull request by nikhita**
**What this PR does / why we need it**: So that tests pass in https://github.com/kubermatic/machine-controller/pull/407.

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -113,5 +113,5 @@ deploy: examples/admission-cert.pem
 check-dependencies:
 	# We need mercurial for bitbucket.org/ww/goautoneg, otherwise dep hangs forever
 	which hg >/dev/null 2>&1 || apt update && apt install -y mercurial
-	$$GOPATH/bin/dep version || (export DEP_RELEASE_TAG=v0.5.0; curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh)
```
Comment (alvaroaleman) [inline Makefile]: To make this less annoying, I'd just add a line `export PATH=$(PATH):$(GOPATH)/bin` at the top of the `makefile`

**Comment by nikhita** [comment]
/test pull-machine-controller-e2e

(running again, want to check something)

**Comment by nikhita** [comment]
Fwiw the lint test passed :tada: :tada: :tada:

**Comment by nikhita** [comment]
e2e test passed!!!! :tada: :tada: :tada:

**Comment by alvaroaleman** [comment]
Omg so awesome :)))))

Can you then disable CircleCI so this PR passes CI? :))))

**Comment by nikhita** [comment]
Done and all tests green :green_heart:

**Comment by nikhita** [comment]
Wohooo! :confetti_ball:

**Pull request merged**